src/virt.c

```diff
@@ -90,6 +108,14 @@
 
 #endif /* LIBVIR_CHECK_VERSION */
 
+/* structure used for aggregating notification-thread data*/
+typedef struct virt_notif_thread_s {
+  pthread_t event_loop_tid;
+  int domain_event_cb_id;
+  pthread_mutex_t active_mutex; /* protects 'is_active' member access*/
+  _Bool is_active;
```

Comment:
it is ok to use bool, true and false now.

Reply:
yes it makes sense - it will look way cleaner than this '_Bool':) This and above reports will be fixed in next update
Thanks for review @rubenk